src/cmd/services/m3dbnode/config/config.go

```diff
@@ -193,8 +193,8 @@ type CommitLogPolicy struct {
 	// enough for almost all workloads assuming a reasonable batch size is used.
 	QueueChannel *CommitLogQueuePolicy `yaml:"queueChannel"`
 
-	// The commit log block size.
-	BlockSize time.Duration `yaml:"blockSize" validate:"nonzero"`
+	// Deprecated. Left in struct to keep old YAMLs parseable.
+	DeprecatedBlockSize *time.Duration `yaml:"blockSize"`
```

Comment:
Interesting, good call - just wondering how we should track some of this stuff so we remove at v1.

Should we get some labels and/or test cases that can be discovered by scanning the repo easily?

Reply:
how about just: `TODO(v1): remove`